examples/configs/devices/Eyelink_1.yml

```diff
@@ -8,4 +8,4 @@ device_model: 'EYELINK CL 6.50  '
 device_firmware: None
 sensor_ids:
 - Eyelink_sens_1
-arg_parser: iout.stim_param_reader.py::DeviceArgs()
+arg_parser: iout.stim_param_reader.py::EyelinkDeviceArgs()
```

Comment:
I think we should eventually move these into the device classes so that device code can truly be a modular unit.

Reply:
I'm not following. There needs to be a reference here to the appropriate parser.  I made the parsers separate from the device classes themselves since pydantic is (by design) pretty finicky. Also, it felt like parsing was a special bundle of functionality rather than something that needed to be in the device classes themselves. Some of those modules are already pretty long.

Reply:
My point is that part of the design goal is to be able to easily drop in a new device or new task without having to edit the existing code base. I.e., we can break up the device code into multiple files, but it is ideally a "bundle" where you just drop the .py files and yaml files into the correct locations and it just works.

So parsing being it's own module makes sense, but anything beyond "default" parameters should specified by the device. (We could support a few useful variants of "default".) Maybe devices can have a submodule that handle specialized parsing for that device if necessary.